src/ui/popup/main-page/sun-moon-icon.tsx

```diff
@@ -0,0 +1,55 @@
+import {m} from 'malevic';
```

Comment:
If it is a full copy-paste, let's use `git mv SRC_PATH DST_PATH` command for `sun-moon-icon.tsx`, `system-icon.tsx` and `watch-icon.tsx` files. This way the history will be preserved. Note that file paths in JS `import` for old design should be corrected.